runtime/src/main/java/org/corfudb/runtime/collections/CorfuTable.java

```diff
@@ -510,28 +528,32 @@ public void clear() {
     @Override
     @Accessor
     public @Nonnull Collection<V> values() {
-        return ImmutableList.copyOf(mainMap.values());
+        return ImmutableList.copyOf(mainMap.values().stream()
+                .map(e -> e.getValue())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)